### PR TITLE
Switch with pulldown resistor and example

### PR DESCRIPTION
**Pull request by BerranRemzi**
I have added functionality for switch connected between a pin and VDD (with pulldown resistor)

### CODE REVIEW

File: examples/ToggleLed_All/ToggleLed_All.ino [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+#include <PinButton.h>
+
+#define BUTTON_PIN            (7)
+
+#define CLICK_LED_PIN         (8)
+#define SINGLE_CLICK_LED_PIN  (9)
+#define DOUBLE_CLICK_LED_PIN  (10)
+#define LONG_CLICK_LED_PIN    (11)
+
+// Create a new button object, listening on pin 5.
```
Comment (poelstra) [inline examples/ToggleLed_All/ToggleLed_All.ino:17]: It became pin 7 in your example.

File: src/PinButton.h [hunk 2]
```diff
@@ -53,17 +53,34 @@ class PinButton : public MultiButton {
...
      * It's recommended to call this method in e.g. loop().
      */
     void update() {
-      MultiButton::update(digitalRead(_pin) == 0);
+      MultiButton::update(digitalRead(_pin) == _pinActiveLevel);
     }
 
   private:
     int _pin;
+    bool _pinActiveLevel = LOW;
```
Comment (poelstra) [inline src/PinButton.h:85]: `digitalRead()` seems to return an `int`, so please make this an `int` as well to prevent compiler warnings in strict modes.

**Review by poelstra — CHANGES_REQUESTED** [review]
Cool! I like the additional example to make it more clear how the different functions can be combined, and indeed the additional ease of using an inverted switch.

Can you please take a look at the two small remarks? Other than that, looks great!

File: examples/ToggleLed_All/ToggleLed_All.ino (hunk 1 above)
Comment (BerranRemzi) [reply examples/ToggleLed_All/ToggleLed_All.ino]: It's done now

File: src/PinButton.h (hunk 2 above)
Comment (BerranRemzi) [reply src/PinButton.h]: It's done now

**Review by poelstra — APPROVED** [review]
Perfect, thanks!

**Comment by poelstra** [comment]
Published as version 1.1.0 :tada:

**Pull request merged**